draft-irtf-cfrg-hpke.md

```diff
@@ -710,11 +711,54 @@ byte strings for public keys.
 | 0x0002 | AES-GCM-256      | 32  | 12  | {{GCM}}      |
 | 0x0003 | ChaCha20Poly1305 | 32  | 12  | {{?RFC8439}} |
 
-# Security Considerations
-
-The general security properties of HPKE are described in
-{{security-properties}}.  In this section, we consider a security issue that may
-arise in practice and an advanced use case.
+# Security Considerations {#sec-considerations}
+
+HPKE makes the following assumptions about its cryptographic
+dependencies.
+
+- Hash: Collision resistance.
+- Extract: Indifferentiable from a Random Oracle.
+- Expand: Behaves as a Pseudorandom Function wherein the first
+argument is the key.
+
+For the DH-based KEM, HPKE also assumes the gap Computational
+Diffie-Hellman (CDH) problem is hard {{S01}}.
```

Comment:
the gap Diffie-Hellman (GDH) problem is hard in the appropriate (sub)group.

I think it is better to cite

```
@InProceedings{Okamoto01,
  author = 	 {Tatsuaki Okamoto and David Pointcheval},
  title = 	 {The Gap-Problems: a New Class of Problems for the Security of Cryptographic Schemes},
  booktitle = {PKC 2001},
  year = 	 2001,
  editor = 	 {K. Kim},
  volume = 	 1992,
  series = 	 lncs,
  pages = 	 {104--118},
  month = 	 feb,
  conflocation = 	 {Cheju Islands, South Korea},
  publisher = "Springer",
}
```

because S01 uses a wrong/uncommon name for the assumption (gap computational), and basically also just cites this one.

Reply:
👍

Reply:
It would also be good to say what property we expect of any non-DH KEMs.  That is, if someone is going to implement this with a new KEM, what do they need to verify about that KEM in order to be confident that HPKE with it is good?

Reply:
Would it be worth a brief note that the DH groups, KDF, and KEMs defined in this document have these properties?